src/ResourceItem.js

```diff
@@ -1,13 +1,34 @@
-import React from 'react';
+import React, { Component } from 'react';
+
+class ResourceItem extends Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			likes: 0
+		};
+		this.Liking = this.Liking.bind(this);
+	  }
+
+	Liking() {
```

Comment:
Liking is a method of component. Should it start from capital letter?

Reply:
no sorry i did by mistake